atomics/performance_tests/kokkos_based/PerfTestAtomics.hpp

```diff
@@ -239,8 +237,7 @@ double test_atomic_perf_random_location(
   int errors = 0;
   Kokkos::parallel_reduce(
       "desul::Tests::PerfRandLoc::Check",
-      Kokkos::RangePolicy<Kokkos::DefaultHostExecutionSpace>(
-          exec_space, 0, result_device.extent(0)),
+      Kokkos::RangePolicy(exec_space, 0, result_device.extent(0)),
```

Comment:
I think this one was correct. `result_device` and `org_dst_values` are host Views.